member-service/src/main/java/com/hedvig/memberservice/commands/SelectNewCashbackCommand.kt

```diff
@@ -0,0 +1,9 @@
+package com.hedvig.memberservice.commands
+
+import org.axonframework.commandhandling.TargetAggregateIdentifier
+import java.util.UUID
+
+class SelectNewCashbackCommand(
+    @TargetAggregateIdentifier val memberId: Long,
```

Comment:
If you want to maintain the same formatting as above
```suggestion
    @TargetAggregateIdentifier 
    val memberId: Long,
```